### PR TITLE
Cherrypy instrumentation docs tests

### PR DESCRIPTION
**Pull request by rahulmukherjee68**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [ ] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [ ] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: instrumentation/opentelemetry-instrumentation-cherrypy/tests/test_cherrypy.py [hunk 1]
```diff
@@ -100,17 +111,25 @@ def healthzz(self):
...
+            
+            @cherrypy.expose
+            def check_header(self):
+                cherrypy.response.headers["custom-test-header-1"]="test-header-value-1"
+                cherrypy.response.headers["custom-test-header-2"]="test-header-value-2"
+                content = {"message": "hello world"}
+                return content
 
-        return cherrypy.tree.mount(CherryPyApp())
+        cherrypy.tree.mount(CherryPyApp())
```
Comment (TheAnshul756) [inline instrumentation/opentelemetry-instrumentation-cherrypy/tests/test_cherrypy.py]: why this change?
Even if we are not using the output we can still return it and ignore it.

**Review by TheAnshul756 — APPROVED** [review]

File: instrumentation/opentelemetry-instrumentation-cherrypy/tests/test_cherrypy.py (hunk 1 above)
Comment (rahulmukherjee68) [reply instrumentation/opentelemetry-instrumentation-cherrypy/tests/test_cherrypy.py]: currently we only need to pass the cherrypyApp object to mount function the return is not used anywhere

Comment (TheAnshul756) [reply instrumentation/opentelemetry-instrumentation-cherrypy/tests/test_cherrypy.py]: yes true. Return object is an CherryPy app so you can return it as it may get used later.

Comment (rahulmukherjee68) [reply instrumentation/opentelemetry-instrumentation-cherrypy/tests/test_cherrypy.py]: ok i am doing that
